src/store/graphql/studies/queries.tsx

```diff
@@ -3,7 +3,7 @@ import { fields } from './models';
 
 export const STUDIES_QUERY = gql`
   query StudiesInformation ($sqon: JSON){
-    study {
+    Studies {
```

Comment:
Maybe prepare the query for paging. Since we know that shortly we will use pagination.